atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -96,108 +94,86 @@ export const createCrashDataMap = isTempRecord => {
     {
-      title: "Fatalities",
-      button: {
-        buttonText: "Reset to CRIS Data",
-        // Define conditions for when button should appear
-        buttonCondition: {
-          dataTableName: "atd_txdot_crashes",
-          dataPath: "apd_human_update",
-          value: "Y",
-        },
-        buttonFieldUpdate: {
-          field: "apd_confirmed_death_count",
-          dataTableName: "atd_txdot_crashes",
-          dataPath: "death_cnt",
-        },
-        buttonConfirm: {
-          confirmHeader: "Are you sure?",
-          confirmBody:
-            "Are you sure you want to revert to the original value from the CRIS database?",
-        },
-        secondaryFieldUpdate: { apd_human_update: "N" },
-      },
+      title: "Injuries",
       fields: {
-        crash_sev_id: {
+        crash_injry_sev_id: {
           label: "Crash Severity",
-          editable: true,
-          uiType: "select",
-          lookupOptions: "atd_txdot__injry_sev_lkp",
-          lookupPrefix: "injry_sev",
-        },
-        vz_fatality_count: {
-          label: "ATD Fatality Count",
-          editable: true,
-          uiType: "text",
-        },
-        death_cnt: {
-          label: "CRIS Death Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
+          uiType: "select",
+          lookupOptions: "injry_sev_lkp",
         },
-        apd_confirmed_death_count: {
-          label: "APD Death Count",
-          editable: true,
-          uiType: "text",
-          secondaryFieldUpdate: { apd_human_update: "Y" },
-        },
-        apd_human_update: {
-          label: "Manually Edited?",
-          editable: false,
-          lookupOptions: "atd_txdot__y_n_lkp",
-          lookupPrefix: "y_n",
-        },
-        law_enforcement_num: {
-          label: "Law Enforcement Number",
-          editable: true,
-          uiType: "text",
-        },
-      },
-    },
-    {
-      title: "Injuries",
-      fields: {
-        non_injry_cnt: {
+        non_injry_count: {
           label: "Not Injured Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        nonincap_injry_cnt: {
+        nonincap_injry_count: {
           label: "Non-incapacitating Injury Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        poss_injry_cnt: {
+        poss_injry_count: {
           label: "Possible Injury Count",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        sus_serious_injry_cnt: {
+        sus_serious_injry_count: {
           label: "Suspected Serious Injury Count",
-          editable: true,
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
           uiType: "text",
         },
-        tot_injry_cnt: {
-          label: "Total Injury Count",
-          editable: true,
+        // tot_injry_cnt: {
```

Comment:
@johnclary is this still a field we want? if so we will need to add it into the `crash_injury_metrics_view`

Reply:
i will check with Xavier—thanks

Reply:
Rose you can remove `tot_injry_cnt`. Per Xavier, you can also remove these from the card:
- Not Injured Count
- Possible Injury Count
- Unknown Injury count